ansible/environments/vagrant/group_vars/all

```diff
@@ -37,6 +37,6 @@ env_hosts_dir: "{{ playbook_dir }}/environments/local"
 
 controller_protocol: "http"
 
-cli_installation_mode: "local"
+cli_installation_mode: "remote"
```

Comment:
Should default to `remote` if you don't want to define this variable. 

https://github.com/apache/incubator-openwhisk/blob/939c3d797b54a870efbb085c14eb7fb4201d8fbb/ansible/group_vars/all#L309

Reply:
I want to defined the variable, just in case for what ever reason the default changes in the top group vars, and is explicit in the environment